captcha/src/main/java/io/nixer/nixerplugin/captcha/security/CaptchaAuthenticationStatusException.java

```diff
@@ -0,0 +1,18 @@
+package io.nixer.nixerplugin.captcha.security;
+
+import org.springframework.security.authentication.AccountStatusException;
+import org.springframework.security.authentication.ProviderManager;
+
+/**
+ * Used to break authentication loop in Spring Security {@link ProviderManager} mechanism when captcha is invalid.
```

Comment:
I would put here a bit more details about the reasons for this exception. E.g. that it actually does not have anything to do with the account status and just leverages catching this particular exception type in particular place of the ProviderManager.